mp/src/game/shared/momentum/weapon/weapon_mom_stickybomblauncher.cpp

```diff
@@ -275,6 +283,14 @@ float CMomentumStickybombLauncher::CalculateProjectileSpeed(float flProgress)
                            MOM_STICKYBOMB_MIN_CHARGE_VEL, MOM_STICKYBOMB_MAX_CHARGE_VEL);
 }
 
+void CMomentumStickybombLauncher::StopChargeSound() 
+{
+    if (m_flChargeBeginTime > 0) // stop only if charging
```

Comment:
I think regardless of whether it was charging, you'll want to `StopWeaponSound(GetWeaponSound("charge"));` as well.